plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastoreConfig.java

```diff
@@ -81,4 +82,16 @@ public CachingHiveMetastoreConfig setMaxMetastoreRefreshThreads(int maxMetastore
         this.maxMetastoreRefreshThreads = maxMetastoreRefreshThreads;
         return this;
     }
+
+    @Config("hive.metastore-partition-cache-enabled")
```

Comment:
`hive.metastore-cache-partitions`

however, we should have `hive.metastore-cache.*` prefix, so maybe  `hive.metastore-cache.cache-partitions`  and rename existing configs in a follow-up.

cc @losipiuk @electrum

Reply:
i suggest `hive.metastore-cache.cache-partitions` and rename existing configs in a follow-up.